gpytorch/mlls/exact_marginal_log_likelihood.py

```diff
@@ -17,6 +19,7 @@ class ExactMarginalLogLikelihood(MarginalLogLikelihood):
 
     :param ~gpytorch.likelihoods.GaussianLikelihood likelihood: The Gaussian likelihood for the model
     :param ~gpytorch.models.ExactGP model: The exact GP model
+    :param ~bool combine_terms (optional): If `False`, the MLL call returns each MLL term separately
```

Comment:
Should probably also describe what happens if there are "other terms" (i.e. that they are added to the return elements)